scripts/release.sh

```diff
@@ -129,11 +129,23 @@ function check_tag_correct() {
 
 # build_release builds the actual release binaries.
 #   arguments: <version-tag> <build-system(s)> <build-tags> <ldflags>
+#              <go-version>
 function build_release() {
   local tag=$1
   local sys=$2
   local buildtags=$3
   local ldflags=$4
+  local goversion=$5
+
+  # Check if the active Go version matches the specified Go version.
+  active_go_version=$(go version | awk '{print $3}' | sed 's/go//')
+  if [ "$active_go_version" != "$goversion" ]; then
```

Comment:
is explicit matching between installed and stipulated go version the intended outcome?

e.g. Developer installs a version >= required go version, the explicit match causes the build to terminate

Reply:
The go version is compiled into the binary. If you don't use the correct version, the hashes won't match.